src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -516,6 +598,29 @@ export class QuickBar extends LitElement {
           color: var(--primary-text-color);
         }
 
+        span.command-category {
+          font-weight: bold;
+          padding: 3px;
+          display: inline-flex;
+          border-radius: 6px;
+          color: black;
+        }
+
+        .command-category.reload {
+          --ha-chip-background-color: #cddc39;
+          --ha-chip-text-color: black;
+        }
+
+        .command-category.navigation {
+          --ha-chip-background-color: var(--light-primary-color);
+          --ha-chip-text-color: black;
```

Comment:
@bramkragten Heads up that I had to force text color to `black` since dark mode changes primary text color to an unreadable shade of gray.

Is there a hard-coded dark shade more appropriate to HA than `black` that I should use?

Reply:
Just hard code the light theme primary text color